app/services/sync_ndbn_members.rb

```diff
@@ -1,69 +1,45 @@
 class SyncNDBNMembers
-  include HTTParty
+  include ServiceObjectErrorsMixin
 
-  NDBN_MEMBERS_PAGE = "https://ndbn.memberclicks.net/member-ids".freeze
+  def initialize(member_file)
```

Comment:
Is there a reason we need this to be an instance? Seems like all it's doing is calling a method and returning a set of error strings. We should be able to do that with a class-level method. Makes it much easier to test.

Reply:
It mainly makes dealing with the errors easier. I have an object to attach the errors to, and then I can check if they exist in the controller.

The other option is having the service return either the parsed values or an error object? I could return something like `parsed_csv, error` and do a check like in Go. But I think using the `ServiceObjectErrorsMixin` makes sense and has precedence since its used in requests, etc.

Reply:
Yeah... I'm not the biggest fan of that mixin. :grin: In this case it doesn't need to return anything other than any possible errors. And... you know, they're errors. :grin: You can just raise an error from this and catch it on the calling side. That's really all that's needed here.

Reply:
You know, I somehow totally blanked that the service was not actually returning anything. Its returning an array of errors now.